docs/development_environment.md

```diff
@@ -4,14 +4,15 @@ title: "Set up Development Environment"
 
 You'll need to set up a development environment if you want to develop a new feature or component for Home Assistant. Read on to learn how to set up.
 
-# Developing with Visual Studio Code + devcontainer
+## Developing with Visual Studio Code + devcontainer
 
 The easiest way to get started with development is to use Visual Studio Code with devcontainers. This approach will create a preconfigured development environment with all the tools you need. This approach is enabled for all Home Assistant repositories.
 
 **Prerequisites**
 
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- [Docker](https://docs.docker.com/install/) (Windows only: requires Windows Pro)
+- [Docker](https://docs.docker.com/install/)
+  - Windows 10 Home requires [WSL 2](https://docs.microsoft.com/windows/wsl/wsl2-install) and the current Edge version of Docker Desktop (see instructions [here](https://docs.docker.com/docker-for-windows/wsl-tech-preview/))
```

Comment:
Should we still mention that Windows Pro can be used too?

Reply:
Yeah that doesn’t come over clear enough. I’ll re write that bit in a little.

Reply:
The way I read it, is that everyone can follow the instructions given @ <https://docs.docker.com/install/> (which for windows pro (among others) takes you to <https://docs.docker.com/docker-for-windows/install/#system-requirements>)

I think it's fine as-is with the extra note for Home, we can remove that when it stops being a preview (since then it will probably be merged into <https://docs.docker.com/docker-for-windows/install/#system-requirements>)

Reply:
I've tidied it up a little. I can see how it might not have been clear to some people not reading in their native language. See https://github.com/home-assistant/developers.home-assistant/pull/467/commits/33b464bcc5655651f3793f39b47284556414ecc9